projects/wireshark/build.sh

```diff
@@ -83,27 +45,4 @@ sed -i '/AC_WIRESHARK_GCC_FORTIFY_SOURCE_CHECK/d' configure.ac
 
 make "-j$(nproc)"
 make install
-
-WIRESHARK_FUZZERS_COMMON_FLAGS="-lFuzzingEngine \
-    -L"$WIRESHARK_INSTALL_PATH/lib" -lwireshark -lwiretap -lwsutil \
-    -Wl,-Bstatic `pkg-config --libs glib-2.0` -pthread -lpcre -lgcrypt -lgpg-error -lz -Wl,-Bdynamic"
-
-for dissector in $FUZZ_DISSECTORS; do
-  generate_fuzzer "${dissector}" -DFUZZ_DISSECTOR_TARGET=\"$dissector\"
-done
-
-for dissector in $FUZZ_IP_PROTO_DISSECTORS; do
-  generate_fuzzer "ip_proto-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"ip.proto\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
-
-for dissector in $FUZZ_TCP_PORT_DISSECTORS; do
-  generate_fuzzer "tcp_port-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"tcp.port\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
-
-for dissector in $FUZZ_UDP_PORT_DISSECTORS; do
-  generate_fuzzer "udp_port-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"udp.port\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
-
-for dissector in $FUZZ_MEDIA_TYPE_DISSECTORS; do
-  generate_fuzzer "media_type-${dissector}" "-DFUZZ_DISSECTOR_TABLE=\"media_type\" -DFUZZ_DISSECTOR_TARGET=\"$dissector\""
-done
+$SRC/wireshark/tools/oss-fuzzshark/build.sh all
```

Comment:
Change to integrate fuzzer to wireshark build system is pending completion and review: https://code.wireshark.org/review/21064/. Original fuzzer is more interactive - it can list all targets, and is using environment variables to select target (instead of prepreocessor defines in oss version).

I am aware that current situation is not perfect, like build fails (https://oss-fuzz-build-logs.storage.googleapis.com/log-472514ce-1a41-4f14-8798-d45084212a65.txt) [fixed upstream], still bigger changes in build system -> more users are affected, it need to be discussed, and changes in tools/oss-fuzzshark/ can be merged fast.